apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/CoverageResourceProvider.java

```diff
@@ -83,6 +86,10 @@ public Coverage read(@IdParam IdType coverageId) {
     if (coverageIdText == null || coverageIdText.trim().isEmpty())
       throw new IllegalArgumentException();
 
+    Operation operation = new Operation(Operation.Endpoint.V1_COVERAGE);
```

Comment:
To check my understanding this will log an `operation` of `V1_COVERAGE` twice. Once here and once in `RequestResponseLoggingFilter`

Reply:
Sort of. The `RequestResponseLoggingFilter` sets the default operation name at the start of each request to `new Operation(Operation.Endpoint.OTHER)`. That gets set in the MDC, which is a single-valued `Map`.

Any handler/endpoint that wants to customize that can override it by creating a more specific `Operation` and calling `publishOperationName()` on it. That publish call overrides the default value in the MDC.

That make sense?

Reply:
it's using `matchByHttpUri` in the default, so I'd expect that to return `V1_COVERAGE` since it's a listed endpoint?

The default logic goes from `requestHttpUri -> Endpoint enum -> requestHttpUri`

Reply:
Ahh, yes -- had forgotten about the matching. That's correct, though it won't have the params -- just the operation name.

Regardless, it won't get logged twice -- the access log is only written to once per response, in `gov.cms.bfd.server.war.RequestResponseLoggingFilter.addToHttpAccessLog()`.

(Being pedantic: note that I said it only gets logged to the *access log* once. The other message/event log will also include whichever values the MDC happens to have at the time of any logging event. It doesn't particularly matter and there's not a lot we can do about it, given SLF4J's API -- short of replacing the MDC entirely, ourselves, which feels like a bad idea.)